### PR TITLE
Fix revert in user edit panel not working

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/8767

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by JanAckermann** [comment]
Do not merge yet, corresponding with ocis team, if the prop `accountEnabled` will be always provided (which wasn't before) we might remove the whole watcher

File: tests/e2e/support/objects/app-admin-settings/users/actions.ts [hunk 1]
```diff
@@ -253,14 +253,24 @@ export const changeUser = async (args: {
...
   value: string
-}): Promise<void> => {
+}): Promise<unknown> => {
   const { page, attribute, value, uuid } = args
   await page.locator(util.format(userInput, attribute)).fill(value)
 
   if (attribute === 'role') {
     await page.locator(util.format(roleValueDropDown, value)).click()
+    return Promise.all([
+      page.waitForResponse(
```
Comment (lookacat) [inline tests/e2e/support/objects/app-admin-settings/users/actions.ts:263]: not sure if promise.all could make problems in case one requests fails

**Review by lookacat — APPROVED** [review]
One thing im not sure about, otherwise fine 👍🏼

File: tests/e2e/support/objects/app-admin-settings/users/actions.ts (hunk 1 above)
Comment (JanAckermann) [reply tests/e2e/support/objects/app-admin-settings/users/actions.ts:263]: this is intended and necessary

Comment (lookacat) [reply tests/e2e/support/objects/app-admin-settings/users/actions.ts:263]: ok then its fine 👍🏼

**Pull request merged**